src/BcPrebidVast.js

```diff
@@ -11,7 +11,7 @@ var _adListManager = require('./AdListManager.js');
 var _prebidCommunicator = require('./PrebidCommunicator.js');
 var _logger = require('./Logging.js');
 
-var PLUGIN_VERSION = '0.4.8';
```

Comment:
Our last released version was 0.4.7 - so we do not need to use 0.4.9

Reply:
My mistake.